modules/zip/zip_packer.h

```diff
@@ -41,10 +41,19 @@ class ZIPPacker : public RefCounted {
 
 	Ref<FileAccess> fa;
 	zipFile zf = nullptr;
+	Vector<uint8_t> write_buffer;
+	String current_password;
+	String current_path;
 
 protected:
 	static void _bind_methods();
 
+#ifndef DISABLE_DEPRECATED
+	Error _start_file_compat_nopw(const String &p_path);
```

Comment:
```suggestion
	Error _start_file_compat_85973(const String &p_path);
```
This is the established format

Reply:
Actually the established format seems to be `_start_file_bind_compat_85973`. We have a few `compat_#####` without `bind_` but it seems to be the exception (and an oversight).

Reply:
You're correct! Wasn't sure (and main point was the number and not a generic description)